test/multiverse/suites/logstasher/Envfile

```diff
@@ -0,0 +1,18 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+instrumentation_methods :chain, :prepend
+
+logstasher_versions = [
+  [nil, 2.6]
+]
+
+def gem_list(logstasher_versions = nil)
+  <<~RB
+    gem 'logstasher'#{logstasher_versions}
+    gem 'activesupport', '7.0.8.4'
```

Comment:
`logstasher` itself depends on `activesupport` and Bundler should automatically bring it in as a dependency. It looks like `logstasher` v1 wants `activesupport` v4+ and `logstasher` v2 wants `activesupport` v5+. What does specifying `'7.0.8.4'` give us? Can we remove this line entirely or do something like `>= 7` if you depend on a major version for your testing?

Reply:
I'm running into testing issues with 7.1, which is why I'm explicity bringing in activesupport. I've changed the pin to look [nicer](https://github.com/newrelic/newrelic-ruby-agent/pull/2559/commits/278c972317055a976cf3a7efc0263ee4c876dd7e) for starters.

Reply:
LogStasher has tested up to Rails 6.1.x, I wonder if there is something going on in Rails 7.1 that is incompatiable.

Reply:
Rails 7.1 introduced a change to their logging system, so that it uses the `ActiveSupport::BroadcastLogger` instead of `ActiveSupport::Logger.broadcast`. I wonder if that's interfering with LogStasher. 

Here's the PR I made a while back to allow our logger instrumentation to work with it: https://github.com/newrelic/newrelic-ruby-agent/pull/2252

Reply:
I just tested a demo app using logstasher instrumentation and Rails 7.1. Everything seemed to work fine, so I think this issue might be with out testing environment.